python/mxnet/ndarray/contrib.py

```diff
@@ -363,3 +362,97 @@ def _func_wrapper(loop_vars):
                 ["  Step %d, shape is %s" % (i, str(x.shape)) for i, x in enumerate(items)]
             ))
     return stacked_outputs, list(loop_vars)
+
+def ifelse(cond, then_func, else_func, inputs):
```

Comment:
Why do we have both cond and inputs? Can we just have cond which could be true/false?

Reply:
to make it consistent with TF?
https://www.tensorflow.org/api_docs/python/tf/cond

Reply:
It may help improve user experience I think.